README.md

```diff
@@ -1,3 +1,70 @@
-# Momento Javascript SDK
+# Momento client-sdk-javascript
 
-Wraps `@momento/generated-types` with higher-level functions.
+:warning: Experimental SDK :warning:
+
+JavaScript SDK for Momento is experimental and under active development.
+There could be non-backward compatible changes or removal in the future.
+Please be aware that you may need to update your source code with the current version of the SDK when its version gets upgraded.
+
+---
+
+<br/>
+
+JavaScript SDK for Momento, a serverless cache that automatically scales without any of the operational overhead required by traditional caching solutions.
+
+<br/>
+
+## Getting Started :running:
+
+### Requirements
+
+- Node version [10.13 or higher](https://nodejs.org/en/download/) is required
+- A Momento Auth Token is required, you can generate one using the [Momento CLI](https://github.com/momentohq/momento-cli)
+
+<br/>
+
+### Installing Momento and Running the Example
+
+Check out our [JavaScript SDK example repo](https://github.com/momentohq/client-sdk-examples/tree/main/javascript)!
```

Comment:
I think you had follow up task to update the Javascript example with the latest version and grab SDK from npm instead of Artifactory?

Reply:
Yup, I already have a branch locally and just need to test with this package.